app/views/catalog/_index_list_default.html.erb

```diff
@@ -22,8 +22,9 @@
     </div>
   <% end %>
   <% if document.children? %>
+    <% @works = ActiveFedora::Base.find(document.id) %>
```

Comment:
We want to use ActiveFedora as little as possible, especially on pages related to the public user experience, as each query can take anywhere from 200ms to multiple seconds.
It might be possible to use `(document.member_ids - document.file_set_ids).count` to get the count of all members of the work, ignoring file sets

Extra note. with valkyrization, `ActiveFedora::Base.find` is replaced by `Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: id)`.
The most common replacements are:
```
ActiveFedora::Base.find(id)
  becomes
Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: id)

curation_concern.update_index
  becomes
Hyrax.index_adapter.save(resource: curation_concern)

curation_concern.save
  becomes
Hyrax.persister.save(resource: curation_concern)
```
And the general class hierarchy from parent to child is:
`Valkyrie::Resource`  =>  `Hyrax::Resource`  =>  `Hyrax::FileSet` (valkyrie version of fileset, instead of our `::FileSet`) || `Hyrax::Work`
`Hyrax::Work` => our work types